lib/push.js

```diff
@@ -253,190 +253,148 @@ module.exports = function (log, db, config) {
 
   return {
 
-    isValidPublicKey: isValidPublicKey,
+    isValidPublicKey,
 
     /**
-     * Notifies all devices that there was an update to the account
+     * Notify devices that there was an update to the account
      *
      * @param {String} uid
+     * @param {Device[]} devices
      * @param {String} reason
+     * @param {Object} [options]
+     *   @param {String[]} [options.includedDeviceIds]
```

Comment:
If the `includedDeviceIds` option is passed, `notifyUpdate` behaves like `pushToDevices` used to. Otherwise it behaves like `pushToAllDevices` used to.